emulation_system/resources/docker/Dockerfile

```
@@ -76,7 +76,9 @@ RUN apt-get update && \
     libboost-test-dev \
     gcc-10 \
     g++-10 \
-    lsb-release  > /dev/null
+    lsb-release \
+    python3.7 \
+    > /dev/null
 
```

It might be helpful to add a symlink to the python executable. But this is not necessarily required
```suggestion
RUN (cd /usr/bin/ && ln -s /usr/bin/python3.7 python)
```